integrations-examples/ftp/pull_mode/pull_from_ftp.py

```
@@ -0,0 +1,251 @@
+        with open(temp_json_path, "w") as f:
+            json.dump(final_result, f, indent=4)
+        
+        # 4. Upload the JSON to the same remote directory as the audio
+        save_file_to_storage(temp_json_path, remote_json_path)
+        
+        # 5. Cleanup local temp JSON file
+        os.remove(temp_json_path)
+        print(f"--- [Thread] Successfully processed and archived JSON for {base_name} ---")
+        print(f"    JSON -> {remote_json_path}")
+
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)